thirdparty/clusterapiprovideribmcloud/v1alpha4/ibmcluster_types.go

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Is there a source we can reference in a comment for `thirdparty/clusterapiprovideribmcloud/v1alpha4` or this actually the source of truth for this API?

Reply:
Right now @enxebre there is no source of truth since the provider itself hasn't been created fully:

I believe that source of truth will ultimately land here:
https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud

(note how there is currently only a v1alpha3 API and there's a lot of fields that I'm not sure will be applicable in our implementation).

My goal with this one is to start with the bare bones minimum to continue to be able to operate in IBM Cloud. That is why there is only the Ready and Control Plane endpoint

Reply:
@relyt0925 Is there a machineController you are planning to use along with this IBMCloudcluster resource?